src/nomad_catalysis/schema_packages/schema.py

```diff
@@ -0,0 +1,424 @@
+#                     [value],
+#                 )
+#             except:
+#                 logger.warn(
+#                     'Something else went wrong when trying setattr for reaction'
+#                 )
+
+
+class Preparation(ArchiveSection):
+    preparation_method = Quantity(
+        type=str,
+        shape=[],
+        description="""
+          Classification of the dominant preparation step
+          in the catalyst synthesis procedure.
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'precipitation',
+                    'hydrothermal',
+                    'flame spray pyrolysis',
+                    'impregnation',
+                    'calcination',
+                    'unknown',
+                ]
+            ),
+            links=['https://w3id.org/nfdi4cat/voc4cat_0007016'],
+        ),
+    )
+
+    preparation_entry_reference = Quantity(
+        type=Process,
+        shape=[],
+        description="""
+        A reference to the entry that contains the details of the preparation method.
+        """,
+        a_eln=dict(component='ReferenceEditQuantity'),
+    )
+
+    preparator = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        The person or persons preparing the sample in the lab.
+        """,
+        a_eln=dict(component='EnumEditQuantity'),
+    )
+
+    preparing_institution = Quantity(
+        type=str,
+        shape=[],
+        description="""
+        institution at which the sample was prepared
+        """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'Fritz-Haber-Institut Berlin / Abteilung AC',
+                    'Fritz-Haber-Institut Berlin / ISC',
+                ]
+            ),
+        ),
+    )
+
+    def normalize(self, archive, logger):
+        super().normalize(archive, logger)
+
+
+class SurfaceArea(ArchiveSection):
+    m_def = Section(
+        label_quantity='method_surface_area_determination',
+        a_eln=ELNAnnotation(label='Surface Area'),
+    )
+
+    surface_area = Quantity(
+        type=np.float64,
+        unit=('m**2/g'),
+        a_eln=dict(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='m**2/g',
+            links=['https://w3id.org/nfdi4cat/voc4cat_0000013'],
+        ),
+    )
+
+    method_surface_area_determination = Quantity(
+        type=str,
+        shape=[],
+        description="""
+          A description of the method used to measure the surface area of the sample.
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'BET',
+                    'H2-TPD',
+                    'N2O-RFC',
+                    'Fourier Transform Infrared Spectroscopy (FTIR)' ' of adsorbates',
```

Comment:
Why the extra quotation marks? This should work the same I think:
```suggestion
                    'Fourier Transform Infrared Spectroscopy (FTIR) of adsorbates',
```

Reply:
if was a leftover from automatic ruff linting fixes, I think

Reply:
thanks for spotting it